bom/pom.xml

```diff
@@ -148,7 +160,7 @@
             <dependency>
                 <groupId>org.jenkins-ci.plugins</groupId>
                 <artifactId>git-client</artifactId>
-                <version>2.8.4</version>
+                <version>3.0.0-beta11</version>
```

Comment:
```suggestion
                <version>2.8.5</version>
```

Reply:
2.8.5 includes the broken tests, so I think @MarkEWaite we need a release for this, or incremental for 2.8.6, or a snapshot will do too just to confirm

Reply:
ah 😢

Reply:
Superseded by #102 I think?

Reply:
yes